functions/ed_admissions_machine_learning.py

```diff
@@ -1,7 +1,9 @@
-
+import numpy as np
+import xgboost as xgb
+from sklearn.compose import ColumnTransformer
 from sklearn.model_selection import TimeSeriesSplit
 from sklearn.metrics import roc_auc_score, log_loss
-
+from sklearn.preprocessing import OneHotEncoder, OrdinalEncoder
```

Comment:
There were various undefined names in this module which I guessed were due to missing imports as changed here.